install-pcf/vsphere/params.yml

```diff
@@ -171,7 +174,8 @@ router_static_ips:      # Comma-separated list of static IPs
 tcp_router_static_ips:  # Comma-separated list of static IPs
 ssh_static_ips:         # Comma-separated list of static IPs
 
-mysql_monitor_email:    # Email address to receive MySQL monitor notifications
+mysql_monitor_email:     # Email address to receive MySQL monitor notifications
+mysql_activity_logging: enable  # Whether to enable internal MySQL server activity logging (enable|disable)
```

Comment:
I have added support of `mysql_activity_logging`. This flag is enabled by default in Elastic Runtime GUI. But I kept running into an error `'mysql_monitor/0 (f4a5b5fb-0868-428a-94c6-6b4322aa5821)' is not running after update. Review logs for failed jobs: replication-canary` in vsphere installs. This error went away after I set `mysql_activity_logging: disable`